doc/changes/changes_1.1.0.md

```diff
@@ -2,6 +2,10 @@
 
 Code name: Parquet support
 
+## Summary
```

Comment:
Maybe you can change the release date right away so you don't need to make additional commit later